setup.py

```diff
@@ -18,7 +18,7 @@
 setup(
     name='prancer-basic',
     # also update the version in processor.__init__.py file
-    version='1.0.39',
+    version='1.0.40',
```

Comment:
@vatsalgit5118  change the version to 1.1.0

Reply:
Updated the version to 1.1.0